decidim-proposals/app/views/decidim/proposals/proposals/show.html.erb

```diff
@@ -47,13 +54,12 @@
   </div>
   <div class="columns mediumlarge-8 mediumlarge-pull-4">
     <div class="section">
-      <%= render partial: "proposal_badge", locals: { proposal: @proposal } %>
+      <%== cell("decidim/proposals/proposal_m", @proposal).badge %>
```

Comment:
I think we should not be using the sized cards directly, everything should go through the main cell